chandra_aca/star_probs.py

```diff
@@ -1022,22 +1053,22 @@ def guide_count(mags, t_ccd, count_9th=False):
     ref_t_ccd = -10.9
     ref_mags0 = (9.0 if count_9th else 9.95) + np.array([0.0, 0.2, 0.3, 0.4])
     ref_mags = {}
-    for t_ccd in np.unique(t_ccds):
+    for t in np.unique(t_ccds):
```

Comment:
Avoid single-letter variable names. Instead you could use `t_ccd_`.

Reply:
OK, I used t_ccd_local in the other spot.

Reply:
In this case `t_ccd_unique` would work and be meaningful.

Reply:
Thanks!  I updated to use `t_ccd_unique` and `t_ccd_` at different places in this function.